core/src/main/java/apoc/export/csv/CsvFormat.java

```diff
@@ -315,10 +302,11 @@ private void writeNodes(SubGraph graph, CSVWriter out, Reporter reporter, List<S
         }
     }
 
-    private void collectProps(Collection<String> fields, Entity pc, Reporter reporter, String[] row, int offset, String delimiter) {
+    private void collectProps(Collection<String> fields, Entity pc, Reporter reporter, String[] row, int offset, boolean useTypes) {
```

Comment:
perché hai aggiunto `useTypes`?

Reply:
Questo non dipende dalla issue, dipende da un bug che ho trovato lavorandoci, cioè se usi la configurazione `useTypes: true` con un valore intero, senza il `fieldNormalized` nella riga 307 non recupera la proprietà.
Visto che ci stavo lavorando l'ho aggiustato in questa pr, ma volendo posso separare

Reply:
Ora l'ho rimosso da qua perché fa parte della #76